rhaptos/cnxmlutils/tests/test_xsl_cnxml_to_html5.py

```diff
@@ -33,6 +34,7 @@ def call_target(self, *args, **kwargs):
             xsl = etree.parse(os.path.join(xsl_dir, 'cnxml-to-html5.xsl'))
             target = etree.XSLT(xsl)
             setattr(self, '_target', target)
+        kwargs['version'] = '"{}"'.format(version)
```

Comment:
If I _don't_ wrap the version in quotes like this, I was seeing errors from XSLT about not being able to parse a user parameter. @philschatz Any thoughts?

Reply:
Yep, adding the extra quotes sounds right. The quotes are how `xsltproc` knows what type to make the param. `true != "true"` for example

Reply:
Yep, adding the extra quotes sounds right. The quotes are how `xsltproc` knows what type to make the param. `true != "true"` for example

Reply:
Ah it seems there's something in `lxml.etree.XSLT` called `strparam` (http://lxml.de/api/lxml.etree.XSLT-class.html#strparamhttp://lxml.de/api/lxml.etree.XSLT-class.html#strparam):

```
Mark an XSLT string parameter that requires quote escaping before passing it into the transformation. Use it like this:

result = transform(doc, some_strval = XSLT.strparam(
    '''it's "Monty Python's" ...'''))
Escaped string parameters can be reused without restriction.
```